content/docs/18-knowledgebase/00-how-to/01-how-to-setup-kube-api-audit-logging.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "How to Configure kube-apiserver audit logging"
+metaTitle: "Configuring the kube-apiserver audit logging featureo"
+metaDescription: "KnowledgeBase with How to"
+icon: "laptop"
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Getting Started with Palette and Auditing Your Kubernetes Cluster Using kube-apiserver Audit Logs
+
+This tutorial is based on the upstream Kubernetes documentation and kube-apiserver source code. It assumes that you have working knowledge of the Kubernetes API.
+
+## What does kubernetes audit logging offer?
+
+Kubernetes auditing provides a security-relevant, chronological set of records documenting the sequence of actions in a cluster. The cluster audits the activities generated by users, by applications that use the Kubernetes API, and by the control plane itself.
+
+Auditing allows cluster administrators to answer the following questions:
+  - what happened?
+  - when did it happen?
+  - who initiated it?
+  - on what did it happen?
+  - where was it observed?
+  - from where was it initiated?
+  - to where was it going?
+
+## What are some scenarios where you would need to enable kube-api audit logging?
+
+- to meet internal security controls
+- to ensure that the actions of individual users can be uniquely traced back to a specific user
+- to debug an issue where resources are being modified by an unknown application
+
+
+## Performance considerations when enabling audit logging
+
+The audit logging feature increases the memory consumption of the API server because some context required for auditing is stored for each request. Memory consumption depends on the audit logging configuration.
+
+
+## Logging all API requests at the metadata level
+
+To start, we will use the [minimal audit policy file](#creating-the-audit-policy-file) to log all requests at the `metadata` level. 
+
+The output will be written to the log file located at `/var/log/kubernetes/audit/audit.log`, which should be sent to and ingested by a logging appliance (ex. Splunk). 
+
+The `/var/log/kubernetes/audit/` directory should be backed by persistent storage to ensure that during an unexpected outage of the node, any unshipped audit logs will not be lost.  
+
+In this example, the logging appliance should be configured to filter the incoming audit logs. 
+
+## Creating the Audit Policy File
+
+On a control-plane node in the target cluster, run the following command to create your audit policy file. 
```

Comment:
```suggestion
On a control-plane node in the target cluster, issue the following command to create your audit policy file. 
```
It's best practice to avoid verbs based on physical traits (accessibility)